app/src/main/java/com/sunasterisk/youtubebackground/data/model/Thumbnail.kt

```diff
@@ -0,0 +1,69 @@
+package com.sunasterisk.youtubebackground.data.model
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class Thumbnail (
+    var default: Default?,
+    var medium: Medium?,
+    var high: High?,
+    var standard: Standard?,
+    var maxRes: MaxRes?
```

Comment:
```suggestion
    var default: ThumbnailSize?,
    var medium: ThumbnailSize?,
    var high: ThumbnailSize?,
    var standard: ThumbnailSize?,
    var maxRes: ThumbnailSize?
```

Reply:
em không dùng inteface nữa do hơi thừa !

Reply:
Mấy cái field đều cùng một kiểu dữ liệu thì tạo một model ra là được rồi.
Nó giống nhau hoàn toàn mà, xoá bớt model ở dưới đi